src/Main.ts

```diff
@@ -544,6 +544,10 @@ export class Main {
         return this.stateStorage?.getState(roomId, eventType, stateKey);
     }
 
+    public async getUserProfile(userId: string): Promise<UserProfile|undefined> {
+        return this.botIntent.getProfileInfo(userId);
```

Comment:
Do we not expose botIntent?, this feels like an odd function to have

Reply:
Indeed, it needs not exist – though it did make testing a lot easier ;) Removed in bbb79af825dd035d4a4b826389b917a7e2bb106e, and adjusted the tests accordingly.